crates/wdk-build/src/utils.rs

```diff
@@ -81,6 +82,42 @@ where
     }
 }
 
+#[derive(Debug, Error, PartialEq, Eq)]
+pub(crate) enum TwoPartVersionError {
+    #[error("Invalid version: {0}. Expected format is 'major.minor'")]
+    InvalidFormat(String),
+    #[error("Error parsing {0} version to 'u32'. Version string: {1}")]
+    ParseError(String, String),
+}
+
+/// Type for versions of the format "major.minor"
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub(crate) struct TwoPartVersion(pub(crate) u32, pub(crate) u32);
```

Comment:
nit: IMO its always clearer to used named args (i.e major and minor) instead of tuple structs

Reply:
I am not sure about that. Tuple structs are a good for for data that is inherently tuple like such as version in this case.

Reply:
but then it becomes ambiguous as to what corresponds with what in the tuple

Reply:
The order of tuple fields corresponds to how you would write the version e.g. 

`"1.5"` -> `TwoPartVersion(1, 5)` 

so it is quite clear which field corresponds to what.